rlclientlib/live_model_impl.cc

```diff
@@ -114,23 +114,23 @@ namespace reinforcement_learning {
     api_status::try_clear(status);
 
     RETURN_IF_FAIL(check_null_or_empty(event_id, context, _trace_logger.get(), status));
-    
+
     float action;
     float pdf_value;
     std::string model_version;
 
     RETURN_IF_FAIL(_model->choose_continuous_action(context, action, pdf_value, model_version, status));
     RETURN_IF_FAIL(populate_response(action, pdf_value, std::string(event_id), std::string(model_version), response, _trace_logger.get(), status));
     RETURN_IF_FAIL(_interaction_logger->log_continuous_action(context, flags, response, status));
-    
+
     if (_watchdog.has_background_error_been_reported())
     {
       RETURN_ERROR_LS(_trace_logger.get(), status, unhandled_background_error_occurred);
     }
 
     return error_code::success;
   }
-    
+
```

Comment:
whitespace

Reply:
added back in. I will remove trailing white spaces on different PR